js/book.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable max-classes-per-file */
+
+// BOOK CLASS represents book
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// STORE CLASS: Handles storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(index) {
+    const books = Store.getBooks();
+
+    books.forEach((book, i) => {
+      if (i === index) {
+        books.splice(index, 1);
```

Comment:
Optional
It is always a better choice to use methods that don't mutate the original data. You can consider using `filter` or `slice`.

Reply:
Thank you 🥲